README.md

```diff
@@ -125,7 +135,34 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+# 1. Create a lamp with 3 wishes remaining and a genie named 'Genie'
+
+lamp = Lamp.create(wishes_remaining: 3)
+genie = Genie.create(name: 'Genie')
+
+# 2. Create a relationship between 'Genie' and the lamp.
+
+class Lamp < ActiveRecord::Base
+  has_one: genie
+end
+
+class Genie < ActiveRecord::Base
+  belongs_to: lamp
+end
+
+# 3. Update the lamp so it only has one wish left.
+
+lamp.update(wishes_remaining: 1)
+
+#   * Oh no... Jafar has Aladdin! Thankfully he's wished to become a genie himself!
+# 4. Create a new Genie named 'Jafar' and put him in a new lamp with 3 wishes left.
+
+bad_lamp = Lamp.create(wishes_remaining: 3)
+bad_lamp.Genie.create(name: 'Jafar') *********** NOT SURE SYNTAX ???? *********
```

Comment:
When referencing associations, you don't use the class name -- e.g., `Genie` (capitalized). If your ActiveRecord classes are set up properly, you can do something like...

```rb
bad_lamp.genie.create( ... )
```

Note that `lamp` is lowercase.